plugins/org.eclipse.papyrus.uml.diagram.sequence.contribution/context/ui/SingleMessage.xwt

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Composite xmlns="http://www.eclipse.org/xwt/presentation"
+	xmlns:ppe="clr-namespace:org.eclipse.papyrus.infra.properties.ui.widgets"
+	xmlns:ppel="clr-namespace:org.eclipse.papyrus.infra.properties.ui.widgets.layout"
+	xmlns:uml="clr-namespace:org.eclipse.papyrus.uml.properties.widgets"
+	xmlns:x="http://www.eclipse.org/xwt">
+	<Composite.layout>
+		<ppel:PropertiesLayout></ppel:PropertiesLayout>
+	</Composite.layout>
+	<Composite>
+		<Composite.layout>
+			<ppel:PropertiesLayout numColumns="2"></ppel:PropertiesLayout>
+		</Composite.layout>
+		<ppe:StringEditor input="{Binding}" property="LWSequenceDiagram:Message:name"></ppe:StringEditor>
+		<ppe:StringEditor input="{Binding}" property="LWSequenceDiagram:Message:label"></ppe:StringEditor>
+	</Composite>
+	<Composite>
+		<Composite.layout>
+			<ppel:PropertiesLayout numColumns="2"></ppel:PropertiesLayout>
+		</Composite.layout>
+		<ppe:ReferenceDialog readonly="true" input="{Binding}" property="LWSequenceDiagram:Message:sendEvent"></ppe:ReferenceDialog>
+		<ppe:ReferenceDialog input="{Binding}" readonly="true" property="LWSequenceDiagram:Message:receiveEvent"></ppe:ReferenceDialog>
+	</Composite>
+	<Composite>
+		<Composite.layout>
+			<ppel:PropertiesLayout numColumns="2"></ppel:PropertiesLayout>
+		</Composite.layout>
+		<ppe:ReferenceDialog input="{Binding}" property="LWSequenceDiagram:Message:signature"></ppe:ReferenceDialog>
+		<ppe:EnumCombo readonly="true" input="{Binding}" property="LWSequenceDiagram:Message:messageSort"></ppe:EnumCombo>
+	</Composite>
+	<Composite>
+		<Composite.layout>
+			<ppel:PropertiesLayout numColumns="2"></ppel:PropertiesLayout>
+		</Composite.layout>
+		<ppe:MultiReference input="{Binding}" property="LWSequenceDiagram:Message:argument"></ppe:MultiReference>
+		<ppe:StringLabel />
```

Comment:
This label that doesn't present any text.  Was it intended to?  Or is it not needed?

Reply:
This is a "trick" also used in Papyrus to prevent the multi-reference control to span two columns. I'll add a comment.

Reply:
Thanks, that will help.  Perhaps there should be a `horizontalSpan="1"` attribute or some such in the XWT but I think that's out of scope.